### PR TITLE
Convert template related services from prisma to drizzle orm

### PR DESCRIPTION
**Pull request by sheharyarahmad842**

### CODE REVIEW

**Comment by sheharyarahmad842** [comment]
I have tried converting template and trip related services. Please let me know if there are issues in it.

File: server/src/controllers/trip/deleteTrip.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { publicProcedure } from '../../trpc';
 import { UnableToDeleteTripError } from '../../helpers/errors';
 import * as validator from '../../middleware/validators/index';
-import { Trip } from '../../drizzle/methods/trip';
+import { Trip } from '../../drizzle/methods/Trip';
```
Comment (raghavyuva) [inline server/src/controllers/trip/deleteTrip.ts:4]: Why this is Trip any errors to change this?

File: server/src/controllers/trip/editTrip.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ export function editTripRoute() {
   return publicProcedure.input(validator.editTrip).mutation(async (opts) => {
     const { id, ...rest } = opts.input;
     const tripClass = new Trip();
-    const trip = await tripClass.update(rest,id);
-    return trip 
+    const trip = await tripClass.update(rest, id);
```
Comment (raghavyuva) [inline server/src/controllers/trip/editTrip.ts:37]: Not sure why you are touching trip related services in this PR

**Review by raghavyuva — CHANGES_REQUESTED** [review]
Refer creating db instance in main pr

File: server/src/controllers/trip/deleteTrip.ts (hunk 1 above)
Comment (sheharyarahmad842) [reply server/src/controllers/trip/deleteTrip.ts:4]: The name of the methods were capitalized so I thought to change "trip" to "Trip". But I'll rename it to trip

File: server/src/controllers/trip/editTrip.ts (hunk 2 above)
Comment (sheharyarahmad842) [reply server/src/controllers/trip/editTrip.ts:37]: Trip and template related services were left so I tried changing them. I asked from Andrew about it.

**Comment by raghavyuva** [comment]
> I have tried converting template and trip related services. Please let me know if there are issues in it.

trip is already converted,

**Comment by sheharyarahmad842** [comment]
I should have pulled the latest code. Now plz tell me what should I do. I don't have much experience with git and github. How can we merge this PR because there can be conflicts.

**Comment by raghavyuva** [comment]
The best thing to do is soft reset head by one commit and make sure to commit back by adding only those files/codes which are relevant to the pr

**Pull request closed without merging**